src/go/rpk/pkg/adminapi/api_cluster.go

```diff
@@ -47,6 +47,13 @@ type PartitionBalancerStatus struct {
 	// CurrentReassignmentsCount is the current number of partition
 	// reassignments in progress.
 	CurrentReassignmentsCount int `json:"current_reassignments_count,omitempty"`
+	// PartitionsPendingForceRecovery Is the number of partitions that are yet
+	// to be force recovered. It's a pointer since not all Redpanda versions
+	// include this parameter.
```

Comment:
```suggestion
	// PartitionsPendingForceRecovery Specifies the number of partitions that are yet
	// to be force recovered. This is a pointer because not all Redpanda versions
	// include this parameter.
```